src/Message/CIMAbstractCustomerProfileRequest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Omnipay\AuthorizeNet\Message;
+
+abstract class CIMAbstractCustomerProfileRequest extends CIMAbstractRequest
+{
+    const VALIDATION_MODE_TEST = 'testMode';
+    const VALIDATION_MODE_LIVE = 'liveMode';
+    const VALIDATION_MODE_NONE = 'none';
+
+    public function setValidationMode($value)
+    {
+        return $this->setParameter('validationMode', $value);
+    }
+
+    public function getValidationMode()
+    {
+        $validationMode = $this->getParameter('validationMode');
+        if ($validationMode !== self::VALIDATION_MODE_NONE) {
+            $validationMode = $this->getDeveloperMode() ? self::VALIDATION_MODE_TEST : self::VALIDATION_MODE_LIVE;
```

Comment:
Fixed in https://github.com/anushr/authorizenet/commit/35fc3a7f151db31dc5d5139534751fb625d94618